### DISCUSSION TITLE
Inconsistency between archived data and latest version

### DISCUSSION THREAD

**Discussion by chestnzu1** [geneontology/helpdesk#348] [Q&A] [answered]
**Please describe your question, idea, or concern.**
Hi guys:
         I am working on some archived data and I just found some of the reviewed human gene annotations were not included in goa_human.gaf file. For example, the annotation file of Sep 1st 2021(http://release.geneontology.org/2021-09-01/annotations/index.html),  there are 296 genes directly annotated by term GO:0007049, while I use QuickGo with filter taxon 9606 and gene product UniProtKB reviewed(swiss-Prot) ,there are 661 annotations. I checked the first two entries, both of which are Swiss-Prot entries but they are not included in the gaf file. According to the description in gaf file, 'the set of protein accessions included in this file is based on UniProt reference proteomes, which provide one protein per gene.
!They include the protein sequences annotated in Swiss-Prot or the longest TrEMBL transcript if there is no Swiss-Prot record.
!If a particular protein accession is not annotated with GO, then it will not appear in this file.' So I hope to know why they are not included
Thanks in advance. 
![fig](https://user-images.githubusercontent.com/67521845/136115718-8fbf6bf2-22d8-4d21-a2a1-81c28bb8f737.PNG)
![fig (2)](https://user-images.githubusercontent.com/67521845/136115774-972de544-a722-4cb8-bbe7-b602129db3af.PNG)

**Comment by suzialeksander** [comment, marked as answer]
Hi @chestnzu1, 

It looks like there was a set of IEAs generated by UniProt on 20210829, including the annotation for UniProtKB:A0AVK6 to GO:0007049 cell cycle.  See geneontology.org/docs/faq/#sometimes-the-number-of-go-annotations-changes-significantly-over-a-short-period-of-time-why

For each GO release, the data is frozen several days prior to allow for the pipeline as well as QC checks, so it seems the IEAs were added on 2021-08-29 just after the 2021-09-01 release had started.  These IEAs will be in the next release.
